core/src/wallet/bitcoin/BitcoinLikeAccount.cpp

```diff
@@ -708,5 +707,195 @@ namespace ledger {
             return _explorer->getFees().callback(getContext(), callback);;
         }
 
+        FuturePtr<api::BitcoinLikeTransaction> BitcoinLikeAccount::getReplayByFeeTransaction(const std::string & hash, const std::shared_ptr<api::Amount> & additionaFeesPerBytes, bool forceBuild) {
+            auto self = getSelf();
+            // Get Transaction hex
+            return _explorer->getRawTransaction(hash).flatMapPtr<api::BitcoinLikeTransaction>(getContext(), [=] (const Bytes &rawTx) -> FuturePtr<api::BitcoinLikeTransaction> {
+                auto tx = std::dynamic_pointer_cast<BitcoinLikeTransactionApi>(
+                        BitcoinLikeTransactionApi::parseRawTransaction(
+                                self->getWallet()->getCurrency(),
+                                rawTx.getContainer(),
+                                self->_currentBlockHeight,
+                                false)
+                );
+
+                // Get input amounts
+                BigInt inputsValue;
+                {
+                    soci::session sql(self->getWallet()->getDatabase()->getPool());
+                    for (auto &input : tx->getInputs()) {
+                        // Check if sequences are lower than 0xffffffff otherwise impossible to use RBF
+                        // TODO: support uints in Djinni
+                        if (!forceBuild && input->getSequence() == std::numeric_limits<uint32_t>::max()) {
```

Comment:
Here you only check "explicit signaling". Are we going to support "Inherited signaling" as well?

Reply:
I think we have to if we want to be RBF compliant.

Reply:
For the moment we are supporting just the `explicit` RBF but we should support the `implicit` as well later